openslides_backend/action/actions/meeting/update.py

```diff
@@ -295,3 +309,12 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
             )
             if not is_superadmin:
                 raise MissingPermission(OrganizationManagementLevel.SUPERADMIN)
+
+    def get_committee_id(self, meeting_id: int) -> int:
+        if not hasattr(self, "__committee_id"):
+            self.__committee_id = self.datastore.get(
+                fqid_from_collection_and_id(self.model.collection, meeting_id),
+                ["committee_id"],
+                lock_result=False,
+            ).get("committee_id", 0)
+        return self.__committee_id
```

Comment:
The datastore already has caching so that the actions don't need to implement it themselves. I'm against doing that here.

Also, access the `committee_id` via `[]` - there should always be one present and if it doesn't, it's better to raise a 500 error instead of using `0` and generating undefined behaviour afterwards.

Reply:
This is still open

Reply:
> I don't want to call 2 times an external service

But you don't, it's an internal call - the datastore adapter caches the result and returns it before you access the database again

Reply:
No, the adapter only caches, if the database access is made with lock_result = True, see https://github.com/OpenSlides/openslides-backend/blob/499cef34dad17578ee0c5141bb164df2be40e709/openslides_backend/services/datastore/cache_adapter.py#L62

Reply:
Then just use `lock_result=True`, locking an fqfield is virtually no overhead. Or, even better, put it in the `prefetch` method (without `lock_result=True`, of course), since that's exactly what it's for.

Reply:
We have problems with locked-fields error and locking such a field is not a good idea. It's a defect in our "action workflow", that our double cache is not able to cache regular data. And the advice to set lock_result = True is even worse!